src/BrokerBot.py

```diff
@@ -53,7 +53,8 @@ class BrokerBot:
     sockets limited to 30
     '''
 
-    def __init__(self, api_key, secret_key, base_url, socket, search_conn):
+    #def __init__(self, api_key, secret_key, base_url, socket, search_conn):
+    def __init__(self, api_key, secret_key, base_url, socket):

```

Comment:
Was removing search_conn here a typo?

Reply:
yea, i was just testing code without it and forgot to change it back